js/parse.js

```diff
@@ -376,7 +377,19 @@ function fillCharacterData(characters, numChars, fields) {
         characters[i].bubblesEquipped = [
             mapLookup(largeBubbleMap, charEquippedBubbles[0]),
             mapLookup(largeBubbleMap, charEquippedBubbles[1])
-        ]
+        ];
+
+        const fieldsPrint = JSON.parse(fields.Print.stringValue);
+        const printData = fieldsPrint.slice(5, fieldsPrint.length); // REMOVE 5 '0' ELEMENTS
+        const chunk = 14; // # OF SAMPLE SLOTS OF ONE CHARACTER
```

Comment:
It might be useful to add a more in-depth comment as to why this is 14. It doesn't have to be too long, just a simple explanation like "7 slots, 5 stored and 2 selected"